### ISSUE TITLE
Empty sourcemaps when using bannerNewLine = true.

### ISSUE THREAD

**Issue by ealmansi** [Comandeer/rollup-plugin-babel-minify#133]
## Steps to reproduce the issue

Add `bannerNewLine: true` to your rollup-plugin-babel-minify config.

## Expected result

Newline should be added after banner, sourcemaps should not be cleared.

## Actual result

Newline is added after banner, but sourcemaps also get cleared. By cleared, I mean that I get the following:

```
{"version":3,"file":"moneybutton.js","sources":[],"sourcesContent":[],"names":[],"mappings":";;;;;;;;;;;;;;;;;;;;"}
```

Disabling this option yields the complete sourcemaps.

## Other details

Looking at the code, the bug seems to be in this line:

https://github.com/Comandeer/rollup-plugin-babel-minify/blob/master/src/index.js#L50

When calling `addNewLine`, the source maps generated by `transform` are completely disregarded.

Version of package: 6.0.0

Node.js & npm versions: 8.11.3 , 5.6.0

Operating system: Ubuntu 18.04.1 LTS

**Comment by Comandeer** [comment]
It will be a little tricky one… First I thought about simply using map generated before adding the new line, but obviously it won't work (or – at best – the whole sourcemap will be shifted by one line). Seems like I need to somehow combined two generated sourcemaps 🤔 There are some npm packages for that and I'm also thinking about [index map](https://sourcemaps.info/spec.html#h.535es3xeprgt). I need to check which one will suit lib better.

**Comment by Comandeer** [comment]
Just released 6.1.1, which should fix this issue!
